app/pages/dashboard/components/table.test.js

```diff
@@ -0,0 +1,103 @@
+import { createTestHarness } from '../../../test-utils/testHarness';
+
+const setup = {
+  component: {
+    name: 'table',
+    path: 'pages/dashboard/components/table.njk',
+  },
+};
+
+const mockContext = {
+  params: {
+    title: 'Manage organisations and users',
+    description: 'Edit existing account information or add new user to an organisation.',
+    columnInfo: [
+      { data: 'Organisation name', link: true },
+      { data: 'ODS code', link: false },
+    ],
+    columnClass: 'nhsuk-grid-column-one-half',
```

Comment:
This will apply the same columnClass for each column? What if we had 3 columns and wanted the first column as half and the other 2 as quarters. 

Do you think it is worth moving this into the object with the column data. Would this give us the flexibility going forward. Also all the column info is contained within columnInfo then?

Reply:
for the use cases we have this will not be the case - we can look at changing it if the need arises